plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1009,15 +1009,25 @@ public void testNullSensitiveTopNPushdown()
     }
 
     @Test
-    public void testPredicateModuloPushdown()
+    public void testArithmeticPredicatePushdown()
     {
-        if (!hasBehavior(SUPPORTS_PREDICATE_MODULO_PUSHDOWN)) {
+        if (!hasBehavior(SUPPORTS_PREDICATE_ARITHMETIC_EXPRESSION_PUSHDOWN)) {
```

Comment:
Do we also push projections as connector expressions? AFAIK we don't. It'd be awesome if we did though - would have side effect of not needing `_EXPRESSION_PREDICATE_PUSHDOWN` and just `_EXPRESSION_PUSHDOWN`.

Reply:
I think I need more explanation on this :)
Maybe some example in existing connectors

Reply:
There is no change required here - I was thinking if we already support pushdown of expressions both in projection (the SELECT clause) as well as predicates (WHERE clause). If we do then it makes sense to test them together if they can be implemented together.

(But reading the code I don't see any rule to push connector expression based projections so my comment has no meaning today).